core/src/main/java/com/linecorp/armeria/common/HttpResponseWriter.java

```diff
@@ -173,7 +192,10 @@ default void respond(HttpStatus status, MediaType mediaType, HttpData content,
 
     /**
      * Writes the specified HTTP response and closes the stream.
+     *
+     * @deprecated Use {@link HttpResponse#of(AggregatedHttpMessage)}.
      */
+    @Deprecated
```

Comment:
Maybe some users will find this method useful when he or she initially created a streaming response and then realized that the stream is actually very small and thus decided to write a fixed response?